src/DropdownMenu/index.ts

```diff
@@ -0,0 +1,3 @@
+export {DropdownMenu} from './DropdownMenu'
+export type {DropdownMenuProps} from './DropdownMenu'
+export {DropdownButton} from './DropdownButton'
```

Comment:
Can we namespace `DropdownButton` under `DropdownMenu` so consumers can use it like this: `DropdownMenu.Button`?

Reply:
Related, from @smockle in https://github.com/primer/components/pull/1135#discussion_r614395895:
> `DropdownButton` could be used with both `DropdownMenu` and `ActionMenu`, right? If so, should it have a name/namespace that doesn’t match one of those exactly (i.e. that is more general)?

Reply:
Reply from @colebemis in https://github.com/primer/components/pull/1135#discussion_r614405677:
> Ah, I see. I didn't realize it was used with `ActionMenu` as well. In that case, I think leaving it as `DropdownButton` is fine 👍